ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -106,22 +104,6 @@ public bool NotificationExists(int NotificationId)
                 .FirstOrDefaultAsync(m => m.NotificationId == NotificationId);
         }
 
-        public async Task<Notification> GetNotificationWithSocialRiskFactorsAsync(int? NotificationId)
```

Comment:
These methods aren't necessary as owned methods are automatically loaded by EF without the need to include them